anda/apps/nekobox/nekobox.spec

```
@@ -0,0 +1,105 @@
+%global _missing_build_ids_terminate_build 0
+%global debug_package %{nil}
+Name: nekobox
+Version: 5.9.20
+Release: 0%{?autorelease}
+Summary: Qt based cross-platform GUI proxy configuration manager (backend: sing-box)
+URL: https://github.com/qr243vbi/nekobox
+License: GPLv3
+
+Source0: %{url}/releases/download/%{version}/nekobox-unified-source-%{version}.tar.xz
+Source1: nekobox.desktop
+Source2: start.sh
+
+BuildRequires: cmake
+BuildRequires: gcc-c++
+BuildRequires: pkgconfig(libcurl)
+BuildRequires: cmake(yaml-cpp)
+BuildRequires: pkgconfig(openssl)
+BuildRequires: cmake(ZXing)
+BuildRequires: cmake(absl)
+BuildRequires: cmake(Qt6)
+BuildRequires: cmake(Qt6Qml)
+BuildRequires: (libboost-devel or boost-devel)
+BuildRequires: thrift
+BuildRequires: (libthrift-devel or thrift-devel)
+BuildRequires: cmake(Qt6Network)
+BuildRequires: cmake(Qt6Svg)
+BuildRequires: cmake(Qt6Linguist)
+BuildRequires: cmake(Qt6Gui)
+BuildRequires: cmake(Qt6Core)
+BuildRequires: cmake(Qt6Widgets)
+BuildRequires: chrpath
+BuildRequires: make
+BuildRequires: sed
+BuildRequires: (ninja or ninja-build)
+BuildRequires: golang >= 1.24
+Requires: nekobox-core
+
+%define main %{name}
+%define core nekobox_core
+
+%package -n nekobox-core
+Summary: %{summary}
+
+%description
+%{summary}.
+
+%description -n nekobox-core
+%{summary}.
+
+%prep
+%autosetup -p1 -n nekobox-unified-source-%{version}
```

```suggestion
%autosetup -n nekobox-unified-source-%{version}
```
Not needed unless you have a patch